rosa_getting_started/rosa-sts-accessing-cluster.adoc

```diff
@@ -0,0 +1,16 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="rosa-sts-accessing-cluster"]
+= Accessing a ROSA cluster
+:context: rosa-sts-accessing-cluster
+
+toc::[]
+
+As a best practice, access your {product-title} (ROSA) cluster using an identity provider (IDP) account. However, the cluster administrator who created the cluster can access it using the quick access procedure.
```

Comment:
Yes, "as a best practice" is needed because there is a shortcut method for the cluster-admin that we document too. Engineering wants them to be aware of both methods and to know which one is recommended.

But, since this was written last fall, I have since learned that we stay away from "as a best practice" and use "it is recommended" instead. So maybe change the wording to the RH standard.